### PR TITLE
Allow specifying SES user creation.

### LINKED ISSUE #15: User creation is unnecessary and brings unneeded dependency
There can be other options for IAM permissions for SeS, for example IAM role, so maybe it will be good if IAM user creation in the scope of this module can be at least disabled

### PR DESCRIPTION
**Pull request by lezavala**
## what
* Add variable `enable_ses_user` (default `true`)
* Allow us to specify whether we want to create the SES IAM user and user policy.

## why
* Creating the user create additional unneeded resources.

## references
* closes #15

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -41,12 +41,13 @@ Create user with permissions to send emails from SES domain
...
   source  = "cloudposse/iam-system-user/aws"
   version = "0.20.2"
+  enabled = var.enable_ses_user
 
   context = module.this.context
 }
 
 data "aws_iam_policy_document" "ses_user_policy" {
-  count = module.this.enabled ? 1 : 0
+  count = var.enable_ses_user ? 1 : 0
```
Comment (osterman) [inline main.tf:50]:
```suggestion
  count = var.ses_user_enabled ? 1 : 0
```

File: main.tf [hunk 2]
```diff
@@ -41,12 +41,13 @@ Create user with permissions to send emails from SES domain
 module "ses_user" {
   source  = "cloudposse/iam-system-user/aws"
   version = "0.20.2"
+  enabled = var.enable_ses_user
```
Comment (osterman) [inline main.tf:44]:
```suggestion
  enabled = var.ses_user_enabled
```

File: main.tf [hunk 3]
```diff
@@ -56,7 +57,7 @@ data "aws_iam_policy_document" "ses_user_policy" {
 
 resource "aws_iam_user_policy" "sending_emails" {
   #bridgecrew:skip=BC_AWS_IAM_16:Skipping `Ensure IAM policies are attached only to groups or roles` check because this module intentionally attaches IAM policy directly to a user.
-  count = module.this.enabled ? 1 : 0
+  count = var.enable_ses_user ? 1 : 0
```
Comment (osterman) [inline main.tf:60]:
```suggestion
  count = var.ses_user_enabled ? 1 : 0
```

File: variables.tf [hunk 4]
```diff
@@ -26,3 +26,9 @@ variable "iam_permissions" {
   description = "Specifies permissions for the IAM user."
   default     = ["ses:SendRawEmail"]
 }
+
+variable "enable_ses_user" {
```
Comment (osterman) [inline variables.tf:30]:
```suggestion
variable "ses_user_enabled" {
```

**Comment by lezavala** [comment]
@osterman are you able to run the rebuild readme?

**Comment by lezavala** [comment]
@osterman fixed the `README.md` issue.

**Review by osterman — APPROVED** [review]

**Pull request merged**